docs/developing.md

```diff
@@ -210,6 +210,33 @@ For example, the Lennard-Jones potential can be parameterized through either the
 %  '''''''''''''
 %  If you define a new class, add new files to autodoc
 
+### Non-bonded methods as implemented in OpenMM
+
+The SMIRNOFF specification describes the contents of a force field, which can be implemented in a number of different ways in different molecular simulation engines.
+The OpenMM implementation provided by the OpenFF Toolkit produces either an `openmm.System` object containing a `openmm.NonbondedForce` object or raises an exception if there is any incompatibility or unimplemented behavior.
+The below table describes these exceptions or which [`NonbondedMethod`](http://docs.openmm.org/latest/userguide/application.html?highlight=ljpme#nonbonded-interactions) is set on the resulting `NonbondedForce`.
```

Comment:
```suggestion
The OpenMM implementation provided by the OpenFF Toolkit either produces an `openmm.System` containing a `openmm.NonbondedForce` object or raises an exception depending on how the non-bonded parameters are specified.
Exceptions are raised when parameters are incompatible with OpenMM (`IncompatibleParameterError`) or otherwise against spec (`SMIRNOFFSpecError`), and also when they are appropriate for the spec but not yet implemented in the toolkit (`SMIRNOFFSpecUnimplementedError`).
This table describes which [`NonbondedMethod`](http://docs.openmm.org/latest/userguide/application.html?highlight=ljpme#nonbonded-interactions) is used in the produced `NonbondedForce`, or else which exception is raised.
```

I had to read this a few times to parse it, this might be a bit clearer. Please correct any incorrect inferences I've made!